config/sample_settings/general.json

```diff
@@ -1,13 +1,20 @@
 {
-    "monitored_accounts": [
+    "tooling_environment": {
+        "Name": "Tooling Account",
+        "AccountId": "323432554",
+        "Region": "eu-central-1"
+    },
+    "monitored_environments": [
```

Comment:
Missing changes to monitoring_groups.json with added "monitored_environment" property

Reply:
Fixed